### PR TITLE
Add a way to remove bundles and versions

### PR DESCRIPTION
**Pull request by RossGoncharuk**
Added a callback to POST `/download` endpoint that would remove bundle after download was successfully completed;
Added script to remove any bundles that are older than `x` amount of seconds (default 300 i.e. 5 minutes) ;
Added script to remove files of versions older then `x` latest versions, default - older than last 10 versions.

### CODE REVIEW

File: routes/routes.js [hunk 1]
```diff
@@ -44,7 +44,13 @@ const appRouter = function (app, gulp) {
       const prebidDownloadFilePath = path.resolve(prebidPath, 'build/dist', fileName);
       console.log(output);
       if(fs.existsSync(prebidDownloadFilePath)) {
-        res.download(prebidDownloadFilePath);
+        res.download(prebidDownloadFilePath, function (err) {
```
Comment (schernysh) [inline routes/routes.js:47]: I'm not an expert here so would like to kindly ask @mkendall07 to chime in.

The goal here is to remove generated file as soon as it is downloaded by the client and not needed anymore.

File: routes/routes.js [hunk 2]
```diff
@@ -44,7 +44,13 @@ const appRouter = function (app, gulp) {
...
+          if (err) {
+            res.end();
+          } else {
+            fs.unlinkSync(prebidDownloadFilePath);
```
Comment (mkendall07) [inline routes/routes.js]: This looks good, the only change I would make would be to have this be an async operations (no reason to hang the client while a file is deleted).  So switch to `fs.unlink`.

**Review by mkendall07 — APPROVED** [review]
Assuming the changes are tested I'm good with this.

File: routes/routes.js (hunk 2 above)
Comment (RossGoncharuk) [reply routes/routes.js]: Will do, thanks!

File: bundleCleanup.sh [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  BUNDLE_LIFE_PERIOD=300
+  echo "deleting bundles older than $BUNDLE_LIFE_PERIOD seconds"
+fi
+
+CURRENT_TIME=$(date +%s)
+
+for DIR in prebid.js/prebid_*;
+  do
+    echo "checking $DIR"
+    BUNDLE_DIR="${DIR}/build/dist/*"
```
Comment (schernysh) [inline bundleCleanup.sh]: This catches all the file under `build/dist`, and all files are being deleted - that's not exactly what we want to happen. Only files named like `prebid.<uuid>.js` in that directory should be subject for deletion.

**Review by schernysh — CHANGES_REQUESTED** [review]

File: bundleCleanup.sh (hunk 3 above)
Comment (RossGoncharuk) [reply bundleCleanup.sh]: Done

Comment (RossGoncharuk) [reply bundleCleanup.sh]: Done

**Review by schernysh — APPROVED** [review]
Looks good, verified.

**Pull request merged**